frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -207,11 +193,17 @@ export const EmbedFrame = ({
             <Separator />
           </Header>
         )}
+        {/**
+         * I put the target for IntersectionObserver right above the parameters container,
+         * so that it detects when the parameters container is about to be sticky (is about
+         * to go out of the viewport).
+         */}
+        <span ref={intersectionObserverTargetRef} />
         {hasVisibleParameters && (
           <ParametersWidgetContainer
             embedFrameTheme={theme}
-            hasScroll={hasInnerScroll}
-            isSticky={isParameterPanelSticky}
+            canSticky={canParameterPanelSticky}
+            isSticky={isFilterSticky}
```

Comment:
These might look a bit confusing.

- `canSticky` was named `isSticky`, so essentially it's never gone, it's still using the same flag, but also renamed as `canParameterPanelSticky`
- `isSticky` is a _new_ flag it used to mean whether or not this component could be sticky, now it means whether or not this component is currently sticky (scrolled).